ledger/participant-state/kvutils/src/main/protobuf/com/daml/ledger/participant/state/kvutils/store/state.proto

```diff
@@ -62,7 +62,9 @@ message DamlCommandDedupValue {
     PreExecutionDeduplicationBounds record_time_bounds = 4;
   }
   string submission_id = 5;
+  google.protobuf.Timestamp expire_at = 6;
```

Comment:
I'm not sure if `expire_at` may be understood in this context, i.e., it is not obvious that it pertains to this particular _deduplication entry_ and not the command it represents.
I'd suggest a bit more explicit field name, such as `prunable_from` or `delete_entry_at` or similar. Alternatively, though less preferred, a short description for the intent may suffice.